Sources/PrimerSDK/Classes/Core/Checkout Components/PrimerHeadlessUniversalCheckoutProtocols.swift

```diff
@@ -136,46 +149,35 @@ public enum PrimerInputElementType: Int {
     
     internal var allowedCharacterSet: CharacterSet? {
         switch self {
-        case .cardNumber:
-            return CharacterSet(charactersIn: "0123456789")
-            
-        case .expiryDate:
-            return CharacterSet(charactersIn: "0123456789")
-            
-        case .cvv:
-            return CharacterSet(charactersIn: "0123456789")
+        case .cardNumber,
+                .expiryDate,
+                .cvv,
+                .otp,
+                .phoneNumber:
+            return CharacterSet.decimalDigits
```

Comment:
I am not very sure about the `decimalDigits`. What does it include? Does it change in other writing systems? Why not using the strictly defined set `CharacterSet(charactersIn: "0123456789")`?

Reply:
Yeah `decimalDigits` also includes representations like the Arabic numeral ones. Do you think it's better to revert that to `0123456789` ? I thought that if we expect decimal numbers strictly being `0123456789` format, we should explain that to the user.